swift/README.md

```diff
@@ -1,12 +1,11 @@
 # Swift on CodeQL
 
-## Warning
+> [!WARNING]
+> The Swift CodeQL package is an experimental and unsupported work in progress.
```

Comment:
I think it's no longer the case, I suggest we pick up wording from [this page](https://codeql.github.com/docs/codeql-language-guides/codeql-for-swift/), and maybe adding the link here?

Reply:
Good point.  Updated.

Reply:
@AlexDenisov I don't think I can merge this until you mark your change request as done.

Reply:
@geoffw0, approved, I didn't know it would be _that_ blocking, TIL 😅

Reply:
Thanks.